### PR TITLE
Ruowan/list operations

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,153 +1,153 @@
...
+		"prepack": "npm run build",
+		"cli": "npm run tsc && node dist/cli.js",
+		"li": "npm install @ts-common/local-install --no-save && local-install"
+	},
+	"engines": {
+		"node": ">=10.11.0"
+	},
+	"jest-junit": {
+		"output": "test-results.xml"
+	}
```
Comment (raych1) [inline package.json:152]: Can you revert indent change?

File: lib/testScenario/gen/staticTestScenarioGenerator.ts [hunk 2]
```diff
@@ -13,11 +13,12 @@ import { RawTestDefinitionFile, TestDefinitionFile, TestResources } from "./../t
 import { TestResourceLoaderOption } from "./../testResourceLoader";
 import { inversifyGetInstance, TYPES } from "./../../inversifyUtils";
 
+type GenerationRule = "put-delete" | "listOperation";
```
Comment (raych1) [inline lib/testScenario/gen/staticTestScenarioGenerator.ts:16]: Can you have consistent rule name? e.g. put-delete-operations, list-operation

File: lib/testScenario/gen/staticTestScenarioGenerator.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ import { RawTestDefinitionFile, TestDefinitionFile, TestResources } from "./../t
 import { TestResourceLoaderOption } from "./../testResourceLoader";
 import { inversifyGetInstance, TYPES } from "./../../inversifyUtils";
 
-type GenerationRule = "put-delete" | "listOperation";
+type GenerationRule = "put-delete" | "operations-list";
```
Comment (raych1) [inline lib/testScenario/gen/staticTestScenarioGenerator.ts:16]: How about change `put-delete` to `operations-putDelete` to be consistent?

Comment (ruowan) [reply lib/testScenario/gen/staticTestScenarioGenerator.ts:16]: emm. The put-delete is for resource put delete. But operations-list is for operationLists

Comment (raych1) [reply lib/testScenario/gen/staticTestScenarioGenerator.ts:16]: Let's rename it as `resource-putDelete`.

**Review by raych1 — APPROVED** [review]

**Pull request merged**